cime_config/config_component.xml

```diff
@@ -199,22 +199,22 @@
     <default_value>UNSET</default_value>
     <values>
       <!-- Use SSP5-8.5 for PI to PDAY so will have data up to the current year available (for WACCM compsets that use PDAY) -->
-      <value compset="^2010_"                       >2010_control</value>
-      <value compset="^2000_"                	    >2000_control</value>
-      <value compset="^1850_"		            >1850_control</value>
+      <value compset="^2010[CE]+_"                  >2010_control</value>
+      <value compset="^2000[CE]+_"                  >2000_control</value>
+      <value compset="^1850[CE]+_"		    >1850_control</value>
       <value compset="^1850_.*_CLM60%[^_]*NOANTHRO" >1850_noanthro_control</value>
       <value compset="^1850_.*_CLM50%[^_]*NOANTHRO" >1850_noanthro_control</value>
-      <value compset="^HIST_"                       >20thC_transient</value>
-      <value compset="^SSP585_"	                    >1850-2100_SSP5-8.5_transient</value>
-      <value compset="^SSP126_"	                    >1850-2100_SSP1-2.6_transient</value>
-      <value compset="^SSP370_"	                    >1850-2100_SSP3-7.0_transient</value>
-      <value compset="^SSP534_"	                    >1850-2100_SSP5-3.4_transient</value>
-      <value compset="^SSP245_"	                    >1850-2100_SSP2-4.5_transient</value>
+      <value compset="^HIST[CE]+_"                  >20thC_transient</value>
+      <value compset="^SSP585[CE]+_"	            >1850-2100_SSP5-8.5_transient</value>
+      <value compset="^SSP126[CE]+_"	            >1850-2100_SSP1-2.6_transient</value>
+      <value compset="^SSP370[CE]+_"	            >1850-2100_SSP3-7.0_transient</value>
+      <value compset="^SSP534[CE]+_"	            >1850-2100_SSP5-3.4_transient</value>
+      <value compset="^SSP245[CE]+_"	            >1850-2100_SSP2-4.5_transient</value>
       <value compset="^HIST" grid="ne0np4"          >1850-2100_SSP2-4.5_transient</value>
-      <value compset="^SSP119_"	                    >1850-2100_SSP1-1.9_transient</value>
-      <value compset="^SSP434_"	                    >1850-2100_SSP4-3.4_transient</value>
-      <value compset="^SSP460_"	                    >1850-2100_SSP4-6.0_transient</value>
-      <value compset="^SSP585_"	                    >1850-2100_SSP5-8.5_transient</value>
+      <value compset="^SSP119[CE]+_"	            >1850-2100_SSP1-1.9_transient</value>
+      <value compset="^SSP434[CE]+_"	            >1850-2100_SSP4-3.4_transient</value>
+      <value compset="^SSP460[CE]+_"	            >1850-2100_SSP4-6.0_transient</value>
+      <value compset="^SSP585[CE]+_"	            >1850-2100_SSP5-8.5_transient</value>
```

Comment:
Just curious whether these compset changes are related to this PR directly or tangentially. I don't mind either way.

Reply:
This is a critical part of what's needed for cesm3_0_beta02 to allow the new C (concentration) and E (emission) compsets. So it's not part of the CAM7 changes, but is needed for this tag.

Reply:
It's actually not clear to me that we will use these flags with cmip6 forcing scenarios.   I would replace the SSP585[CE]+ with SSP585[CE]?

Reply:
Yep, I'll do that @jedwards4b

Reply:
Done